File: Web/RMA-Member-Portal/memberportal-app/src/app/shared/models/roleplayer-contact-information.ts
import { ContactInformationTypeEnum } from "../enums/contact-information-type-enum";

export class RolePlayerContactInformation {
    rolePlayerContactInformationId: number;
    rolePlayerContactId: number;
    contactInformationType: ContactInformationTypeEnum;
    isDeleted: boolean;
    createdBy: string;
    createdDate: Date;
    modifiedBy: string;
    modifiedDate: Date;
}

File: Service/ClientCare/RMA.Service.ClientCare.Contracts/Entities/RolePlayer/RolePlayerContactInformation.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.ClientCare.Contracts.Entities.RolePlayer
{
    public class RolePlayerContactInformation
    {
        public int RolePlayerContactInformationId { get; set; } // RolePlayerContactInformationId (Primary key)
        public int RolePlayerContactId { get; set; } // RolePlayerContactId
        public ContactInformationTypeEnum ContactInformationType { get; set; } // ContactInformationTypeId
        public bool IsDeleted { get; set; } // isDeleted
        public string CreatedBy { get; set; } // CreatedBy (length: 50)
        public System.DateTime CreatedDate { get; set; } // CreatedDate
        public string ModifiedBy { get; set; } // ModifiedBy (length: 50)
        public System.DateTime ModifiedDate { get; set; } // ModifiedDate

    }
}
